crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -350,6 +352,51 @@ impl Display for SuiParsedTransactionResponse {
     }
 }
 
+#[allow(clippy::large_enum_variant)]
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+pub enum SuiExecuteTransactionResponse {
+    ImmediateReturn {
+        tx_digest: TransactionDigest,
+    },
+    TxCert {
+        certificate: SuiCertifiedTransaction,
+    },
+    // TODO: Change to CertifiedTransactionEffects eventually.
+    EffectsCert {
+        certificate: SuiCertifiedTransaction,
+        effects: SuiCertifiedTransactionEffects,
```

Comment:
Can we add the `timestamp_ms` and `parsed_data` here as well, as in https://github.com/MystenLabs/sui/blob/11aa1f68ed8eb7df667785992dd79669991d294c/crates/sui-json-rpc-types/src/lib.rs#L310 ? This makes migration easier for CLI and the wallet

Reply:
does gateway return a timestamp for txn execution today?

Reply:
> does gateway return a timestamp for txn execution today?

No. 

> For this return struct, we are looking to add timestamp, it needs some extra work though

got it, `parsed_data` is higher pri than `timestamp_ms` in this case. Let me take a look